package.json

```diff
@@ -93,7 +93,8 @@
     "swr": "^0.4.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.10.2",
+    "@babel/core": "7.13.10",
```

Comment:
I think you may unlock the version, yarn is better at securing us against unexpected updates nowadays